chirp/wxui/clone.py

```diff
@@ -118,12 +119,14 @@ def _add_grid(label, control):
         hbox2.Add(cancel, flag=wx.RIGHT)
 
         vbox = wx.BoxSizer(wx.VERTICAL)
-        vbox.Add(grid, proportion=0, flag=wx.ALIGN_CENTER|wx.TOP|wx.BOTTOM,
+        vbox.Add(grid, proportion=0, flag=wx.ALIGN_CENTER_HORIZONTAL|wx.TOP,
                  border=20)
-        vbox.Add(self.gauge, flag=wx.EXPAND, border=10, proportion=1)
+        vbox.Add(self.gauge, flag=wx.EXPAND|wx.LEFT|wx.RIGHT, border=10, proportion=0)
         vbox.Add(wx.StaticLine(panel), flag=wx.EXPAND|wx.TOP, border=10)
         vbox.Add(hbox2, proportion=0, flag=wx.ALIGN_RIGHT|wx.ALL, border=10)
         panel.SetSizer(vbox)
+        self.Layout()
+        self.Center()
```

Comment:
I don't know why you're making these changes. The clone dialog looks identical before/after them AFAICT. Can you please write more descriptive commit messages for things like this to explain what you're doing and why?

Reply:
Without these changes the widgets in the clone dialog overlapping and it is a little mess, as far as I can see in two different linux systems. Also the wx.gauge is not visible without them. Currently on pyhton 3.9.2, wxpython 4.0.7 and wxwidgets 3.0.5

Reply:
Hmm, I don't see that at all on my linux machine. Python 3.9, wxwidgets 3.0.4 and wxpython 4.0.7 here. Maybe it's a theming thing? Can you post a screenshot so I can see? But, it looks the same to me so I guess it's okay.